src/web/src/app/space/service.ts

```diff
@@ -186,36 +198,49 @@ export class SpaceService {
 	}
 
 	join(spaceId: string, userId: string): Observable<any> {
-		return this.http.post(this._apiUrl + "/invitation", {
+		return this.apiSpaceService.handlerInvitationPost("user", {
```

Comment:
Подобные XUser хидеры перетрутся проксёй. Передаём пустые значения:
```suggestion
		return this.apiSpaceService.handlerInvitationPost('', {
```

Reply:
Поправил везде.